components/atoms/ProfileInfoSection.js

```diff
@@ -0,0 +1,39 @@
+import propTypes from 'prop-types'
+
+export default function ProfileInfoSection(props) {
+  return (
+    <>
+      <h3 className="border-b border-gray-300 font-bold">{props.title}</h3>
```

Comment:
Double check your heading levels, this needs to be an h2 (I also don't see an h1, but this isn't where that would go)
```suggestion
      <h2 className="border-b border-gray-300 font-bold">{props.title}</h2>
```